### PR TITLE
Jc new room description

### PR DESCRIPTION
**Pull request by jcode89**
Created functionality to display a description of the room you are in when you start that particular level.

### CODE REVIEW

**Comment by phillipgray** [comment]
"But where are the tests?" said the testing goat.

**Comment by jcode89** [comment]
Haha. Touché. 

Jonathan Colón

**Comment by jcode89** [comment]
So do I have to add a description to all of those JSON files that you use for the tests?

**Comment by jcode89** [comment]
Okay @jamalhansen @phillipgray I added some unit and functional tests, however I was having trouble with the functional test. For some reason the description of the room is not appending to the `self.printed` list created in `helpers.py`

**Comment by jamalhansen** [comment]
Obey the testing goat

File: resources/level_1.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "room": "level 1",
     "size": 18,
     "exit_text": "You have unlocked the exit and proceed through it into a dark and ominous hallway",
+	"description": "The room is dark and damp. You can see a giant padlock on the other door and in the distance a big chest.",
```
Comment (jamalhansen) [inline resources/level_1.json]: Not sure what happened here, but I'm gonna be picky and ask you to un-indent these, because Python :)

**Pull request merged**